api/internal/wrappy/factory.go

```diff
@@ -76,11 +78,100 @@ func (k *WNodeFactory) Hasher() ifc.KunstructuredHasher {
 }
 
 func (k *WNodeFactory) MakeConfigMap(
-	kvLdr ifc.KvLoader, args *types.ConfigMapArgs) (ifc.Kunstructured, error) {
-	panic("TODO(#WNodeFactory): implement MakeConfigMap")
+	ldr ifc.KvLoader, args *types.ConfigMapArgs) (ifc.Kunstructured, error) {
+	rn, err := k.makeConfigMap(ldr, args)
+	if err != nil {
+		return nil, err
+	}
+	return FromRNode(rn), nil
+}
+
+func (k *WNodeFactory) makeConfigMap(
+	ldr ifc.KvLoader, args *types.ConfigMapArgs) (*yaml.RNode, error) {
+	rn := yaml.MustParse(`
+apiVersion: v1
+kind: ConfigMap
+`)
+	err := applyGeneratorArgs(rn, ldr, args.GeneratorArgs)
+	return rn, err
 }
 
 func (k *WNodeFactory) MakeSecret(
-	kvLdr ifc.KvLoader, args *types.SecretArgs) (ifc.Kunstructured, error) {
-	panic("TODO(#WNodeFactory): implement MakeSecret")
+	ldr ifc.KvLoader, args *types.SecretArgs) (ifc.Kunstructured, error) {
+	rn, err := k.makeSecret(ldr, args)
+	if err != nil {
+		return nil, err
+	}
+	return FromRNode(rn), nil
+}
+
+func (k *WNodeFactory) makeSecret(
+	ldr ifc.KvLoader, args *types.SecretArgs) (*yaml.RNode, error) {
+	rn := yaml.MustParse(`
+apiVersion: v1
+kind: Secret
+`)
+	err := applyGeneratorArgs(rn, ldr, args.GeneratorArgs)
+	if 1+1 == 2 {
```

Comment:
is this intentional?

Reply:
yes, intentional.  binary encoding not done yet,
so if the `--enable_kyaml` flag is flipped, kustomize does nothing but throw errors.

All the tests for MakeSecret are written to demand an error right now.

I wanted to get this directionally correct before dealing with binary encoding